compare_weather_sources.py

```
@@ -93,14 +93,14 @@ async def compare_weather_sources():
     print()
     print("Commands to download 2024 data:")
     print("-" * 80)
-    print("# JMA AMeDAS (1,286 stations, 10-minute data):")
-    print("python download_weather_parallel.py --source jma --country JP --start 2024-01-01 --end 2024-12-31 --max-locations 100")
+    print("# JMA AMeDAS (1,300+ stations, 10-minute data - recent only):")
+    print("python download_weather_incremental.py --source jma --country JP --start [recent-date] --end [today] --max-locations 100")
```

Pylint (3.3.7)

[convention] Line too long (133/100)

(C0301)